defaults/main.yml

```diff
@@ -40,6 +40,17 @@ os_security_init_prompt: true
 # Require root password for single user mode. (rhel, centos)
 os_security_init_single: false
 
+# Apply ufw defaults
+ufw_manage_defaults: true
+
+# Disable IPT_SYSCTL in /etc/default/ufw
```

Comment:
Can you add the default value `/etc/ufw/sysctl.conf` as a comment here so one knows what the os-default is?